recipes/sacremoses/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "sacremoses" %}
+{% set version = "0.0.35" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "1e84da95cbdbfc8bbc148c0feb5a4737c5d9757071b79e8c1129e449410e410a"
+
+build:
+  noarch: python
+  number: 0
+  entry_points:
+    - sacremoses=sacremoses.cli:cli
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - click
```

Comment:
You probably don't need all of these, `python` and `pip` is likely enough for the `host` requirements.